pkg/manifest/manifest.go

```diff
@@ -393,6 +400,9 @@ type Step struct {
 }
 
 func (s *Step) Validate(m *Manifest) error {
+	if s == nil {
+		return errors.New("found an empty step. probably you forgot to mention it")
+	}
```

Comment:
To Reviewer: I thought this is right place to check for `nil`. Let me know your opinion

Reply:
It seems like the natural spot, and it works with your test data, so 👍